src/imitation/scripts/config/parallel.py

```diff
@@ -21,6 +25,21 @@ def config():
     "named_configs": [],
     "config_updates": {},
   }  # `config` argument to `ray.tune.run(trainable, config)`
+  s3_bucket = None  # Used to create default `upload_dir` if not None.
+
+
+@parallel_ex.config
+def ray_upload_dir(inner_experiment_name, s3_bucket):
+  if s3_bucket is not None:
+    upload_dir = "s3://{}".format(
+      osp.join(s3_bucket, inner_experiment_name, util.make_unique_timestamp()))
```

Comment:
This is a very flat directory structure. Fine if your workflow is: find the Sacred config.json, look up the upload_dir, grab from S3. But you won't be able to find the experiment you want just from looking at the directory.

Reply:
I've only been looking at the most recent S3 uploads during prototyping (copy and paste to cd into the the final output of `ls /tmp/shwang-chai/`). I can imagine it being hard to distinguish between all the uuids later. Probably at this point I will support adding a prefix to the `util.make_unique_timestamp`.